File: app/src/main/java/com/example/ikastyle/Dao/ShoesGearDao.java
package com.example.ikastyle.Dao;

import androidx.room.Dao;
import androidx.room.Query;

import com.example.ikastyle.Entity.HeadGear;
import com.example.ikastyle.Entity.ShoesGear;

import java.util.List;

@Dao
public interface ShoesGearDao {
    @Query("SELECT * FROM MAST_SHOES_GEAR WHERE language_code = :languageCode ORDER BY name")
    List<ShoesGear> getShoesGearList(int languageCode);
}

File: app/src/main/java/com/splatool/ikastyle/model/data/dao/ShoesGearDao.kt
package com.splatool.ikastyle.model.data.dao

import com.splatool.ikastyle.model.data.entity.ShoesGear
import androidx.room.*

@Dao
interface ShoesGearDao {
    @Query("SELECT * FROM MAST_SHOES_GEAR WHERE language_code = :languageCode ORDER BY name")
    suspend fun getShoesGearList(languageCode: Int): List<ShoesGear>
}